keymap_drawer/parse.py

```diff
@@ -74,6 +76,10 @@ def mapped(key: str) -> str:
             return LayoutKey(tap=mapped(m.group(2).strip()), hold=m.group(1).strip())
         if m := self._lt_re.fullmatch(key_str):
             return LayoutKey(tap=mapped(m.group(2).strip()), hold=f"L{m.group(1).strip()}")
+        if m := self._osm_re.fullmatch(key_str):
+            return LayoutKey(tap=mapped(m.group(1).strip()), hold="sticky")
+        if m := self._osl_re.fullmatch(key_str):
+            return LayoutKey(tap=mapped(m.group(1).strip()), hold="sticky")
```

Comment:
```suggestion
            return LayoutKey(tap=f"L{m.group(1).strip()}", hold="sticky")
```
Since we know this is a layer index, we can skip the `mapped` conversion and add the "L" prefix similar to `MO` key handling on the first case.